src/tests/build.sh

```diff
@@ -100,6 +100,7 @@ build_Tests()
     export __CopyNativeTestBinaries
     export __Priority
     export __CreatePerfmap
+    export __HotColdSplitting
```

Comment:
We don't need to add Hot/Cold Splitting here. This is just for the normal build and we want it in the test builds, i.e. in `src/tests/build.sh`. As for how to handle the logic of how the test scripts are generated, take a look at `src/tests/Common/helixpublishwitharcade.proj` and `CLRTest.Execute.Bash.targets`. It's all defined there. It's hard to explain from here so feel free to reach out to me offline for more specific guidance on how they work :)

Reply:
Actually never mind. I thought this was `./build.sh`. Leaving the comment open for visibility on the explanation on the next steps.